.reaction/scripts/setup

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
The path for this file is different than the other projects, where it's at `bin/setup`. Is there a reason to put it in a nested directory? I'd favor a consistent path if possible.

Reply:
@ticean Meteor eagerly loads all folders that aren't hidden, so that's why I changed it. However, as long as nobody puts a `.js` file in `bin/setup`, it probably won't cause problems. Might make Meteor a tiny bit slower having another folder to watch, but it's probably negligible, so I could try going back to bin/setup.